aiosonic/__init__.py

```diff
@@ -222,11 +222,11 @@ async def read_chunks(self) -> AsyncIterator[bytes]:
 
     def __del__(self):
         # clean it
-        if self.chunked and not self.chunks_readed:
-            loop = None
-            if self.connection:
-                loop = get_loop()
-                loop.create_task(self.connection.release())
+        if self.connection and self.connection.blocked:
+            if self.connection.writer:
+               self.connection.writer._transport.abort()
```

Comment:
these two stuffs can be inside connection class

Reply:
Oh, I see. Good catch indeed